.gitignore

```
@@ -0,0 +1,4 @@
+*.py[cod]
+*/*/*/__pycache__/
+*/*/__pycache__/
+*/__pycache__/
```

It's enough to go for it
```suggestion
__pycache__/
```